Content.Server/_Goobstation/StationEvents/GameDirectorSystem.cs

```diff
@@ -0,0 +1,576 @@
+using System.Linq;
+using Content.Server._DV.StationEvents.Components; // DeltaV - ignore events from the game directors
+using Content.Server._Goobstation.StationEvents.Components;
+using Content.Server._Goobstation.StationEvents.Metric;
+using Content.Server.Administration.Logs;
+using Content.Server.Chat.Managers;
+using Content.Server.GameTicking.Rules;
+using Content.Server.GameTicking.Rules.Components;
+using Content.Server.StationEvents;
+using Content.Server.StationEvents.Components;
+using Content.Shared._Goobstation.CCVar;
+using Content.Shared._Goobstation.StationEvents; // DeltaV - shared the prototypes for our tests
+using Content.Shared._Goobstation.StationEvents.Metric; // DeltaV - shared the prototypes for our tests
+using Content.Shared.CCVar;
+using Content.Shared.Database;
+using Content.Shared.EntityTable;
+using Content.Shared.EntityTable.EntitySelectors;
+using Content.Shared.GameTicking.Components;
+using Content.Shared.Ghost;
+using Content.Shared.Humanoid;
+using Content.Shared.Prototypes;
+using Content.Shared.Tag;
+using JetBrains.Annotations;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Enums;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Timing;
+using YamlDotNet.Core.Tokens;
```

Comment:
Using directive is unnecessary
```suggestion
using System.Linq;
using Content.Server._DV.StationEvents.Components; // DeltaV - ignore events from the game directors
using Content.Server._Goobstation.StationEvents.Components;
using Content.Server.Administration.Logs;
using Content.Server.Chat.Managers;
using Content.Server.GameTicking.Rules;
using Content.Server.GameTicking.Rules.Components;
using Content.Server.StationEvents;
using Content.Server.StationEvents.Components;
using Content.Shared._Goobstation.CCVar;
using Content.Shared._Goobstation.StationEvents; // DeltaV - shared the prototypes for our tests
using Content.Shared._Goobstation.StationEvents.Metric; // DeltaV - shared the prototypes for our tests
using Content.Shared.Database;
using Content.Shared.GameTicking.Components;
using Content.Shared.Ghost;
using Content.Shared.Humanoid;
using Content.Shared.Prototypes;
using Content.Shared.Tag;
using JetBrains.Annotations;
using Robust.Server.Player;
using Robust.Shared.Configuration;
using Robust.Shared.Enums;
using Robust.Shared.Player;
using Robust.Shared.Prototypes;
using Robust.Shared.Random;
using Robust.Shared.Timing;
```